UnityProject/Assets/Scripts/Items/Others/MixingBowl.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Mirror;
+public class MixingBowl : NetworkBehaviour, IServerInventoryMove
```

Comment:
Does this need to be a network behaviour? Try switching to mono and then if that works remove the using mirror.

Reply:
it does, i tried monobehavior and it doesn't work

Reply:
Ah cool.